apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -1,37 +1,71 @@
+import CurriculumCatalogCard from '@cdo/apps/templates/curriculumCatalog/CurriculumCatalogCard';
+import {translatedCourseOfferingDurations} from '@cdo/apps/templates/teacherDashboard/CourseOfferingHelpers';
 
-class CurriculumCatalog extends Component {
-  static propTypes = {
-    curriculaData: PropTypes.arrayOf(curriculaDataShape)
-  };
+const CurriculumCatalog = ({curriculaData}) => (
+  <>
+    <div className={style.catalogHeader}>
+      <HeaderBanner
+        headingText={i18n.curriculumCatalogHeaderTitle()}
+        subHeadingText={i18n.curriculumCatalogHeaderSubtitle()}
+        short={false}
+        imageUrl={CourseCatalogIllustration01}
+      />
+    </div>
+    <div className={style.catalogContentContainer}>
+      <div className={style.catalogContent}>
+        {/*TODO [MEG]: calculate and pass in duration and translated from backend */}
+        {curriculaData
+          .filter(curriculum => !!curriculum.grade_levels)
+          .map(
+            ({
+              key,
+              image,
+              display_name,
+              grade_levels,
+              school_subject,
+              cs_topic
+            }) => {
+              // TODO [MEG]: We are currently assuming if there are grade levels, there are at least two
+              // grades and the list is in ascending order.
+              const gradeLevelArray = grade_levels.split(',');
+              const durationPossibilities = Object.keys(
+                translatedCourseOfferingDurations
+              );
 
-  render() {
-    return (
-      <>
-        <div className={style.catalogHeader}>
-          <HeaderBanner
-            headingText={i18n.curriculumCatalogHeaderTitle()}
-            subHeadingText={i18n.curriculumCatalogHeaderSubtitle()}
-            short={false}
-            imageUrl={CourseCatalogIllustration01}
-          />
-        </div>
-        <div className={style.catalogContentContainer}>
-          <div className={style.catalogContent}>
-            {this.props.curriculaData.map(curriculum => (
-              <li key={curriculum.key}>{curriculum.display_name}</li>
-            ))}
-          </div>
-        </div>
-      </>
-    );
-  }
-}
+              return (
+                <CurriculumCatalogCard
+                  key={key}
+                  imageSrc={image}
+                  courseDisplayName={display_name}
+                  duration={
+                    durationPossibilities[
+                      Math.floor(Math.random() * durationPossibilities.length)
+                    ]
+                  } // TODO [MEG] actually pass in this data
+                  youngestGrade={gradeLevelArray[0]}
+                  oldestGrade={gradeLevelArray[gradeLevelArray.length - 1]}
+                  subjects={school_subject?.split(',')}
+                  topics={cs_topic?.split(',')}
+                  isTranslated={!!Math.round(Math.random())} // TODO [MEG]: actually pass in this data
```

Comment:
is this here so you can test both ways? this could be a little confusing to someone trying to repro a bug, so I'd prob recommend hard-coding it, and just using the react browser tool to toggle the prop value if you want to be able to see it change in the browser. no need to go back through drone just for this, though.

Reply:
It was more for "easy" screenshots that would include the various options. But yes, I'll remove this in the next PR.